cstore_fdw.c

```diff
@@ -672,6 +681,55 @@ CStoreTable(Oid relationId)
 
 
 /*
+ * DistributedTable checks if the given relationId is the OID of a distributed table,
+ * which may also be a cstore_fdw table, but in that case COPY should be handled by
+ * Citus.
+ */
+static bool
+DistributedTable(Oid relationId)
+{
+	bool isDistributedTable = false;
```

Comment:
maybe use distributedTable instead of isDistributedTable ? we do not use -is- prefix in similar cases